.docker/qgis/scripts/enable_nzgbplugin

```diff
@@ -2,6 +2,9 @@
 
 CONFIGFILE=/root/.local/share/QGIS/QGIS3/profiles/default/QGIS/QGIS3.ini
 
+# Install startup.py
+cp /scripts/startup.py /root/.local/share/QGIS/QGIS3/startup.py
```

Comment:
Did you forget to `git add` the `startup.py` script ? I don't see it in teh diff screen

Reply:
Yip. Added